black_nb/cli.py

```diff
@@ -128,6 +150,8 @@ def cli(
     check: bool,
     include: str,
     exclude: str,
+    extend_exclude: str,
```

Comment:
After adding the `validate_regex` callback, the type of `include` and exclude should be `Pattern[str]`. The type of `extend_exclude` should be `Optional[Pattern[str]]`. The type of `stdin_filename` should be `Optional[str]`.

Reply:
updated the type as well :+1: